internal/controller/datadogagent/feature/otelcollector/feature.go

```diff
@@ -68,6 +74,29 @@ func (o *otelCollectorFeature) ID() feature.IDType {
 }
 
 func (o *otelCollectorFeature) Configure(dda metav1.Object, ddaSpec *v2alpha1.DatadogAgentSpec, _ *v2alpha1.RemoteConfigConfiguration) feature.RequiredComponents {
+	var agentImageName string
+	agentVersion := images.AgentLatestVersion
+	if nodeAgent, ok := ddaSpec.Override[v2alpha1.NodeAgentComponentName]; ok {
+		if nodeAgent.Image != nil {
+			agentImageName = nodeAgent.Image.Name
+			agentVersion = common.GetAgentVersionFromImage(*nodeAgent.Image)
+		}
+	}
+	supportedVersion := utils.IsAboveMinVersion(agentVersion, "7.67.0-0", apiutils.NewBoolPointer(true))
+	if !supportedVersion && agentImageName == "" {
+		o.incompatibleImage = true
+		o.logger.Error(errIncompatibleImage, "OTel Agent Standalone image requires agent version 7.67.0 or higher. Update the Agent version or use the agent image with -full tag instead.",
+			"current_version", agentVersion)
+		return feature.RequiredComponents{}
+	}
+
+	if strings.HasSuffix(agentVersion, "-full") && agentImageName == "" {
+		o.incompatibleImage = true
+		o.logger.Error(errIncompatibleImage, "OTel Agent Standalone image does not support full tag. Update the Agent to version without -full tag, or use the agent image with -full tag instead.",
```

Comment:
I don't think this error message is really helpful/clear, I think the intent is to ask users to use a FQDN override, e.g. `override.nodeAgent.image.name: registry.domain/agent:version-full` so all containers in the Agent pod use the `-full` image ?